### PR TITLE
Untracked: fix zoom in small multiples

### PR DESCRIPTION
**Pull request by chschan**
This fixes an issue where zooming in small multiples with shared axes didn't get applied to all panels

### CODE REVIEW

File: theSrc/test/snapshots/travis/master/zoom/shared_axes_after_zoom-snap.png
Comment (chschan) [inline theSrc/test/snapshots/travis/master/zoom/shared_axes_after_zoom-snap.png:1]: I'm not sure why the title is moving like that, but I manually checked this on my browser and I didn't see it. It seems to only occur on the old chrome browser puppeteer is using

File: theSrc/test/snapshots/travis/master/stateInteractions/smallmultiples_hide_label-snap.png
Comment (chschan) [inline theSrc/test/snapshots/travis/master/stateInteractions/smallmultiples_hide_label-snap.png:1]: I had to change the test for toggling labels to an example that didn't use shared axes. But manually testing on my browser, toggle still works fine with the shared axes.

File: theSrc/test/snapshots/travis/master/zoom/shared_axes_after_zoom-snap.png
Comment (JustinCCYap) [reply theSrc/test/snapshots/travis/master/zoom/shared_axes_after_zoom-snap.png:1]: This is known flakiness. Perhaps remove the title from the test?

**Review by JustinCCYap — APPROVED** [review]
Good to see code simplification. I didn't know about `matches`.

**Pull request merged**